playground/api/v1/api.proto

```diff
@@ -314,6 +314,17 @@ message GetSnippetResponse {
   Complexity complexity = 4;
 }
 
+// GetMetadataRequest represents request for runner metadata
+message GetMetadataRequest {}
+
+// GetMetadataResponse contains metadata about the runner
+message GetMetadataResponse {
+  string runner_sdk = 1;
+  string build_commit_hash = 2;
+  string build_commit_timestamp = 3;
```

Comment:
```suggestion
  int64 build_commit_seconds_since_epoch = 3;
```
?

In Dart core these are called `millisecondsSinceEpoch`, `microsecondsSinceEpoch`, etc. to avoid confusion since people nowadays expect varying units of timestamp.